### ISSUE TITLE
Pdbfixer does note save file.

### ISSUE THREAD

**Issue by ismailhakkiakgun** [pandegroup/pdbfixer#134]
Hello I am using Pdbfixer on Ubuntu 14.04. When ever I try to save fixed structure I got following error. How can I fix this problem?

```
Exception happened during processing of request from ('127.0.0.1', 54895)
Traceback (most recent call last):
  File "/home/ismail/anaconda2/lib/python2.7/SocketServer.py", line 599, in process_request_thread
    self.finish_request(request, client_address)
  File "/home/ismail/anaconda2/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/home/ismail/anaconda2/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/home/ismail/anaconda2/lib/python2.7/BaseHTTPServer.py", line 340, in handle
    self.handle_one_request()
  File "/home/ismail/anaconda2/lib/python2.7/BaseHTTPServer.py", line 328, in handle_one_request
    method()
  File "/home/ismail/anaconda2/lib/python2.7/site-packages/pdbfixer/uiserver.py", line 29, in do_POST
    self.invokeCallback(parameters)
  File "/home/ismail/anaconda2/lib/python2.7/site-packages/pdbfixer/uiserver.py", line 41, in invokeCallback
    callback[path](parameters, self)
  File "/home/ismail/anaconda2/lib/python2.7/site-packages/pdbfixer/ui.py", line 139, in saveFilePageCallback
    output.write("REMARK   1 PDBFIXER FROM: %s\n" % fixer.source)
TypeError: unicode argument expected, got 'str'
```

**Comment by peastman** [comment]
Looks like an incompatibility with Python 2.  Let me look into it.

**Comment by peastman** [comment]
I just sent a PR with the fix: #135.  Could you test it and verify that it fixes the problem for you?
